python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -505,6 +506,7 @@ std::string getQIRLL(const std::string &name, MlirModule module,
   auto context = cloned.getContext();
 
   PassManager pm(context);
+  pm.addPass(cudaq::opt::createLambdaLiftingPass());
```

Comment:
Are we using cc callables in the python bridge?

Reply:
The following example fails without the lambda lifting pass so I think we do: 
```python
@cudaq.kernel
def test():
    def k():
        q1 = cudaq.qvector(2)
    k()

print(test)
print(cudaq.translate(test, format='qir'))
```

Reply:
I added a test to make sure this example does not fail (for qir profiles)